src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -127,6 +127,9 @@ static ExperimentalFeature()
                 new ExperimentalFeature(
```

Comment:
@adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-windows`
XmlCommand DRT basic functionality Tests.Import-Clixml StopProcessing should succeed
```powershell
Expected 'Stopped', but got Completed.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\XMLCommand.Tests.ps1: line 77
77: 		$ps.InvocationStateInfo.State | Should -Be "Stopped"
```